### ISSUE TITLE
core-js 2 deprecated

### ISSUE THREAD

**Issue by sam-s4s** [angular/angular#34643]
# 🐞 bug report

### Is this a regression?

 Yes and no - it's the underlying npm dependencies that have changed to cause this problem, not angular itself.

### Description
When creating a new angular package, or doing an npm install, we now receive a warning that core-js 2 is deprecated and we should be using 3.

## 🔬 Minimal Reproduction
ng new
(create a new angular 8 project)

## 🔥 Exception or Error
<pre><code>
npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
</code></pre>

## 🌍  Your Environment

**Angular Version:**
<pre><code>
Angular CLI: 8.3.21
Node: 10.15.1
OS: win32 x64
Angular: 8.2.14
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.803.21
@angular-devkit/build-angular     0.803.21
@angular-devkit/build-optimizer   0.803.21
@angular-devkit/build-webpack     0.803.21
@angular-devkit/core              8.3.21
@angular-devkit/schematics        8.3.21
@angular/cli                      8.3.21
@ngtools/webpack                  8.3.21
@schematics/angular               8.3.21
@schematics/update                0.803.21
rxjs                              6.4.0
typescript                        3.5.3
webpack                           4.39.2
</code></pre>

**Anything else relevant?**

**Comment by alfaproject** [comment]
Angular 8 doesn't require `core-js`. If you do then you can update it yourself, if you don't then you can remove it.

**Comment by internalsystemerror** [comment]
Yarn gives a bit more info
```
warning @angular-devkit/build-angular > istanbul-instrumenter-loader > istanbul-lib-instrument > babel-generator > babel-runtime > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
```

**Comment by sam-s4s** [comment]
@alfaproject yes, this is from a project that does not have core-js as a dependency. As @internalsystemerror points out, it's part of angular.

Though possible this is more of a cli issue, given the `@angular-devkit/build-angular` package...
